sbin/setup

```diff
@@ -22,3 +22,32 @@ $ROOT/sbin/system-setup.py $SETUP_ARGS
 if [[ $VERBOSE == 1 ]]; then
 	python3 -m pip list
 fi
+
+# This entire section purpose is to make sure that on amazon_linux_2
+# we link to openssl1.1. To achieve it, we uninstall 'openssl-devel'
+# which was already installed by this setup file and then install 'openssl11-devel'.
+# Notice that we can not install them both at the same time because they conflict with
+# each other. When we remove this 'setup' file from the build steps we should perform
+# this installation on '.install/amazon_linux_2.sh' (same as we do for RediSearch).
+OS_TYPE=$(uname -s)
+if [[ $OS_TYPE = 'Darwin' ]]
+then
+    OS='macos'
+else
+    VERSION=$(grep '^VERSION_ID=' /etc/os-release | sed 's/"//g')
```

Comment:
Ideally I would source /etc/os-release then deal with it. It's key/value pairs and meant for shell scripts.

Reply:
We are gone drop this code anyway when we refactor the build system and this is how it is currently done on search so I wanted to keep it the same.
Is it critical?